website/sidebars.js

```diff
@@ -221,25 +221,27 @@ const sidebarSettings = {
       type: "category",
       label: "Deploy dbt jobs",
       collapsed: true,
+      link: {type: "doc", id: "docs/deploy/deployments"},
       items: [
-        "docs/deploy/deployments",
+        "docs/deploy/about-state",
```

Comment:
can we order this below the Deploy with dbt Cloud section? felt a little weird that this was given the top spot in the menu